pkg/restore/restore.go

```diff
@@ -454,6 +459,70 @@ func (ctx *context) execute() (Result, Result) {
 		}
 	}
 
+	// TODO: Re-order this logic so that CRs can be prioritized in the main loop, rather than after.
+
+	// Refresh and resolve based on CRDs added to the API server from the above restore loop.
+	ctx.discoveryHelper.Refresh()
+	newResources, _ := prioritizeResources(ctx.discoveryHelper, ctx.resourcePriorities, ctx.resourceIncludesExcludes, ctx.log)
```

Comment:
add error-check

Reply:
Thanks, I hadn't had one since it was a quick hack, but if it's going in for the point release, definitely need it.

Reply:
yep, i figured :)